apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,323 +77,266 @@ export function InstructorQuestionSettings({
+                  !resLocals.course.example_course
+                    ? html`
+                        <button
+                          type="button"
+                          class="btn btn-xs btn-secondary mr-2"
+                          id="changeQidButton"
+                          data-toggle="popover"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Change QID"
+                          data-content="${renderEjs(
+                            __filename,
+                            "<%= include('../partials/changeIdForm') %>",
+                            {
+                              id_label: 'QID',
+                              buttonID: 'changeQidButton',
+                              id_old: resLocals.question.qid,
+                              ids: qids,
+                              ...resLocals,
+                            },
+                          )}"
+                          data-trigger="click"
+                        >
+                          <i class="fa fa-i-cursor"></i>
+                          <span>Change QID</span>
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
+              </div>
+              <div class="form-group">
+                <div>Topic</div>
+                <div name="topic">
+                  ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                    topic: resLocals.topic,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Tags</div>
+                <div name="tags">
+                  ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
+                    tags: resLocals.tags,
+                  })}
```

Comment:
Now that we're giving ourselves more vertical space, maybe we render this as a list group instead? Something like this:

<img width="414" alt="Screenshot 2024-04-17 at 11 50 43" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/8bfe95d3-ae15-40bd-a1e8-daa2ede89cdc">

Though we'd want to align the name/description in different columns, sort of like GitHub's "labels" page:

<img width="1239" alt="Screenshot 2024-04-17 at 12 09 52" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/e7eac0d4-02a0-46b8-8957-298b114dc514">

Bootstrap's grid system will probably come in handy for that.

Reply:
In [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7), updated the "Tags" section to look like the following: <img width="829" alt="Screenshot 2024-04-18 at 5 19 43 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/114321556/ceafa153-229f-483d-8dfc-7bb0921db39e">